app/views/shared/feedback_forms/_form.html.erb

```diff
@@ -22,13 +22,13 @@
         <div class="form-group row">
           <%= label_tag(:name, 'Your name', class:"col-md-3 col-form-label") %>
           <div class="col-md-9">
-            <%= text_field_tag :name, "", class:"form-control", required: true %>
+            <%= text_field_tag :name, current_user&.display_name, class:"form-control", required: true %>
```

Comment:
Another nicety here would be `autocomplete: 'name'` Then the browser may help the user by autopopulating.